open_event/models/speaker.py

```diff
@@ -71,3 +73,19 @@ def serialize(self):
                 'position': self.position,
                 'country': self.country,
                 'sessions': session_data  }
+
+    def ensure_socialLinks(self):
+        """convert usernames in social network fields to full links"""
+        self.twitter = self._ensure_socialLink('https://twitter.com', self.twitter)
+        self.facebook = self._ensure_socialLink('https://www.facebook.com', self.facebook)
+        self.github = self._ensure_socialLink('https://github.com', self.github)
+        self.linkedin = self._ensure_socialLink('https://www.linkedin.com/in', self.linkedin)
+
+    def _ensure_socialLink(self, website, link):
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://www.codacy.com/app/triplez/open-event-orga-server/file/2406529482/issues/source?bid=3233864&fileBranchId=3233886#l84)